GitUtils.ps1

```diff
@@ -159,13 +159,19 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
 
                 $indexAdded.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexAdded))))
                 $indexModified.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexModified))))
-                $indexModified.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexRenamed | Select-Object -ExpandProperty Old))))
+                $indexRenamedOld = $cacheResponse.IndexRenamed.Old

```

Comment:
[Member enumeration](https://blogs.msdn.microsoft.com/powershell/2012/06/13/new-v3-language-features/) but it is only available in PS v3 and higher.   To make that work on PS v2, use:
```powershell
$indexRenamedOld = $cacheResponse.IndexRenamed | Foreach-Object { $_.Old }
```